CHANGELOG.md

```diff
@@ -16,7 +20,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Fixed
  - Invalid user dictionaries are handled more robustly ([#28])
 
-[Unreleased]: https://github.com/envato/zxcvbn-ruby/compare/v1.0.0...HEAD
+[1.1.0]: https://github.com/envato/zxcvbn-ruby/compare/v1.0.0...HEAD
```

Comment:
Should it be `1.1.0` instead of `HEAD` here?

```suggestion
[1.1.0]: https://github.com/envato/zxcvbn-ruby/compare/v1.0.0...1.1.0
```

Reply:
Yes. Well spotted. Thanks.

Reply:
This is what it should be...
```suggestion
[1.1.0]: https://github.com/envato/zxcvbn-ruby/compare/v1.0.0...v1.1.0
```